lib/network.go

```diff
@@ -4772,9 +4916,173 @@ type TransactionSpendingLimit struct {
 	// BuyingCreatorPKID || SellingCreatorPKID to number of
 	// transactions
 	DAOCoinLimitOrderLimitMap map[DAOCoinLimitOrderLimitKey]uint64
+
+	// ===== ENCODER MIGRATION UnlimitedDerivedKeysMigration =====
+	// IsUnlimited field determines whether this derived key has no spending limit.
```

Comment:
```suggestion
	// IsUnlimited field determines whether this derived key can perform any transaction on behalf of its owner and does not require a no spending limit.
```
let's be more clear about what `IsUnlimited` means